src/Views/Shared/AlertListCompact.elm

```diff
@@ -0,0 +1,22 @@
+module Views.Shared.AlertListCompact exposing (view)
+
+import Alerts.Types exposing (AlertGroup)
+import Html exposing (Html, ul)
+import Html.Attributes exposing (class)
+import Views.Shared.AlertCompact
+
+
+view : AlertGroup -> Html msg
+view ag =
+    let
+        alerts =
+            List.concatMap
+                (\b ->
```

Comment:
I think the current way is more descriptive and easier to read. Not a strong opinion. Happy to change it to your suggestion though?!